src/components/registration/DisplayUser.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import { TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
+
+const DisplayUser = (props) => {
```

Comment:
Destructure prop object, that way code will be much cleaner and readable. Check `prop-types` npm package for typechecking.